scoring/fitness.metta

```diff
@@ -1,4 +1,6 @@
 
+! (import! &self metta-moses:utilities:general-helpers)
```

Comment:
One last comment, we don't need this import. It should work fine as long as you are not calling the function in this file. 
```suggestion

```